src/celluloid-application.c

```diff
@@ -317,16 +317,26 @@ local_cmdline_handler (	GApplication *gapp,
 	gchar **argv = *arguments;
 	CelluloidApplication *app = CELLULOID_APPLICATION(gapp);
 	g_clear_pointer(&app->mpv_options, g_free);
+	gboolean mpv = FALSE;
+	GArray *options = g_array_new(TRUE, FALSE, sizeof(gchar *));
 	while(argv[i])
 	{
-		if(g_strcmp0(argv[i], "--") == 0)
-		{
-			app->mpv_options = g_strjoinv(" ", &argv[i + 1]);
-			argv[i] = 0;
-			break;
+		if((mpv && g_str_has_prefix(argv[i], "--")) || (g_strcmp0(argv[i], "--") == 0)){
+			if(mpv)
+				g_array_append_val(options, argv[i]);
+			else
+				mpv = TRUE;
+
+			for(gint j = i; argv[j]; j++)
+				argv[j] = argv[j + 1];
+		}else{
+			i++;
```

Comment:
For consistency, always use Allman-style indentation. Also, always put braces around the body of if/for even if it's only a single line.

Reply:
For some reason, I was completely sure that the project uses K&R or something like that. I don't know why.